sdk/highlight-node/package.json

```diff
@@ -27,7 +27,7 @@
 		"access": "public"
 	},
 	"dependencies": {
-		"@prisma/instrumentation": "^5.22.0",
+		"@prisma/instrumentation": "^6.2.1",
```

Comment:
@Vadman97 would we actually want to make this more relaxed and lower the version to something like `>=5.0.0`? Also, do you think it makes sense to convert this to a peer dependency?

Reply:
@ccschmitz would that require us doing a dynamic import in the SDK, conditional on the `@prisma/instrumentation` package being available?

Reply:
If it's a peer dependency, wouldn't this just make it less likely to run into version conflicts? I _think_ the way this would work is that if they don't have the package installed it would install it for them, and it would reach for the latest version. If they do have it installed already it would use that version. By referencing an earlier version I think we would be more lenient and avoid potential conflicts with earlier versions of the package being installed in the consumer.

Reply:
hmm, maybe i don't understand the difference between a peer dependency of `"@prisma/instrumentation": ">=5.0.0"` and a normal dependency of `"@prisma/instrumentation": ">=5.0.0"`, as once i moved this to a peer dep i have a typescript error because `@highlight-run/node` does not have `@prisma/instrumentation` installed

`TS2307: Cannot find module @prisma/ instrumentation or its corresponding type declarations.`

Reply:
Oh, right... I think the `peerDependencies` behavior is different between yarn and npm. I think npm >=7 installs them by default, but yarn does not.